### PR TITLE
Add runbook for NoReadyVirtController alert

### PR DESCRIPTION
**Pull request by sradco**
Signed-off-by: Shirly Radco <sradco@redhat.com>

### CODE REVIEW

File: docs/runbooks/NoReadyVirtController.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# NoReadyVirtController
+
+## Meaning
+
+From a high-level perspective the virt-controller has all the cluster wide virtualization functionality.
```
Comment (iholder-redhat) [inline docs/runbooks/NoReadyVirtController.md:5]: Would change `virt-controller has all the cluster wide virtualization functionality` to `virt-controller is responsible for all the cluster wide virtualization functionality`.

**Review by iholder-redhat — COMMENTED** [review]
Thanks @sradco for your work!

What about fetching logs? It's one of the best way to debug and diagnose problems.
You can maybe add:
```
kubectl logs -n kubevirt virt-controller-XYZ
```

Also - is there a relatively known use case in which virt-controller can be unready for a long time? Maybe we can list some probable root causes?

**Comment by djfjeff** [comment]
To make this runbook compatible with both upstream and downstream, would you consider exporting the namespace and using it after (for example, kubevirt for upstream, openshift-cnv for downstream)? I see in other runbook the following :

- Set the environment variable NAMESPACE

```
export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
```

- Then use it in other commands, for example

`kubectl -n $NAMESPACE get pods -l kubevirt.io=virt-api`

File: docs/runbooks/NoReadyVirtController.md (hunk 1 above)
Comment (sradco) [reply docs/runbooks/NoReadyVirtController.md]: @iholder-redhat Updated. Please review.

**Comment by sradco** [comment]
> To make this runbook compatible with both upstream and downstream, would you consider exporting the namespace and using it after (for example, kubevirt for upstream, openshift-cnv for downstream)? I see in other runbook the following :
> 
> * Set the environment variable NAMESPACE
> 
> ```
> export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
> ```
> 
> * Then use it in other commands, for example
> 
> `kubectl -n $NAMESPACE get pods -l kubevirt.io=virt-api`

@djfjeff Updated. Please review.

**Comment by djfjeff** [comment]
@sradco this looks good to me!

**Pull request merged**